spyder/plugins/editor/widgets/codeeditor/lsp_mixin.py

```diff
@@ -1199,6 +1207,10 @@ def _apply_document_edits(self, edits):
 
             # End text insertion
             cursor.endEditBlock()
+            
+            # Restore breakpoints
+            if breakpoints is not None and breakpoints:
```

Comment:
```suggestion
            if breakpoints:
```

After a second thought, I think we can simply write `if breakpoints` here because it cover both cases (i.e. that `breakpoints` is not `None` and has some content).